docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# HydraDX Code of Conduct
+
+## Our Pledge
```

Comment:
I feel like this captures everything we need. Maybe we'd like to add one more sentence 

"If you feel harassed or attacked for any of those, please contact the administrators." 

And keep just the link to attribution of the covenant.